app/partials/content/Accordion.html

```diff
@@ -9,6 +9,7 @@
             
             <span class="accordion-section-title-suffix" ng-bind="titleSuffixes[$index]" math-jax></span>
             <div class="ru_accordion_name" style="float: left;">
+                <span class='ru_level' ng-if="c.level"></span>
```

Comment:
In 1c0cac47f4db975b773c77bc908dcbacedd8d633, I was a little overzealous with my undoing of changes on the GraphSketcher branch. Thanks for spotting and fixing it @bh412 !